.github/workflows/pre-release.yml

```diff
@@ -0,0 +1,26 @@
+name: pre-release
+
+on:
+  push:
+    tags:
+      - '*.*.*'
+
+jobs:
+  pre-release:
+    name: Pre Release
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Build & test
+        run: |
+          echo "done!"
+
+      - uses: marvinpinto/action-automatic-releases@latest
+        with:
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          automatic_release_tag: "latest"
+          prerelease: true
+          title: "Development Build"
+          files: |
```

Comment:
The `prerelease` parameter defaults to `true`, and our build doesn't have any `.jar` files or a `LICENSE.txt` file. This raised an error in the Action output, but that didn't show as a failure. We should remove them.
<img width="433" alt="Screen Shot 2021-06-24 at 12 57 41 PM" src="https://user-images.githubusercontent.com/177431/123310530-cad08b80-d4eb-11eb-8159-b3b7e9f9ca51.png">